Source/WebKit/GPUProcess/graphics/RemoteImageBuffer.cpp

```diff
@@ -120,6 +120,8 @@ void RemoteImageBuffer::getShareableBitmap(WebCore::PreserveResolution preserveR
         auto backendSize = m_imageBuffer->backendSize();
         auto logicalSize = m_imageBuffer->logicalSize();
         auto resultSize = preserveResolution == WebCore::PreserveResolution::Yes ? backendSize : m_imageBuffer->truncatedLogicalSize();
+        if (resultSize.isEmpty())
+            return std::nullopt;
```

Comment:
How is related to this PR?

Reply:
I was hitting an assert without it so I added it so the fuzzer can still pass through